js/main.js

```diff
@@ -296,6 +317,11 @@
     savebutton.value = 'Save';
     ui.show(document.querySelector('#savebutton'));
     shareTarget.captureImage();
+    video.pause();
+    if (localstream && typeof localstream.getTracks === "function") {
```

Comment:
Localstream is assigned from `stream`  in the `getUserMedia` callback and initially defined as `null`. This is how we actually stop the camera itself.